doc-content/enterprise-only/project-data/create-branches-proc.adoc

```diff
@@ -0,0 +1,20 @@
+[id='create-branches-proc']
+
+= Creating branches
+You can create new branches in {CENTRAL} and name them whatever you like. Initially, you will only have the default *master* branch. When you create a new branch for a project, you are making a copy of the selected branch and all of its assets. You can make changes to the project on the new branch without impacting the original *master* branch version.
+
+.Procedure
+. Log in to {CENTRAL} and click *Menu* -> *Design* -> *Projects*.
+. Click on the project to create the new branch, for example the *Mortgage Process* sample project.
+. Click *master* -> *Add Branch*.
++
+.Create the new branch menu
+image::getting-started/new-branch.png[Creating the new branch menu]
+
+. Type `testBranch1` in the *Name* field and select *master* in the *From* field of the *Add Branch* window.
+. Click *Add*.
++
+.Add the new branch window
+image::getting-started/test-branch.png[Adding the new branch testBranch1]
```

Comment:
I wonder when we do screenshots and when we don't, is there guidance somewhere that we follow?

Reply:
I tend to wing it. In using the UI, I go with my own instincts as to some of the less obvious menus etc. Really not sure about the hard and fast rules. I just know that reader prefer them.